<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginForm</class>
 <widget class="QDialog" name="loginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Twitterak</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>UserName : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtloginuser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtloginpass">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="UserNameLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>UserName : </string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Password : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtusername">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtpass">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtmobile">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="MobileLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Mobile : </string>
    </property>
   </widget>
   <widget class="QLabel" name="CountryLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Country : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtcountry">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Account Type</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Organisation</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Anonymouse</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Personal</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);
background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtboss">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="BossLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Boss /Company :</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Name : </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>BossLable</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>txtboss</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CountryLable</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>txtcountry</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MobileLable</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>txtmobile</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>NameLable</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>txtname</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>BossLable</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>CountryLable</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>MobileLable</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>NameLable</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>txtname</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>txtmobile</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>txtcountry</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>txtboss</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>txtboss</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>txtcountry</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>txtmobile</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>txtname</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>BossLable</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>CountryLable</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MobileLable</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>NameLable</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>txtboss</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>txtcountry</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>txtboss</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>txtcountry</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
